docs/source/api_reference/forecasting.rst

```diff
@@ -341,6 +341,17 @@ Transformer (deep learning) based forecasters
     LTSFDLinearForecaster
     LTSFNLinearForecaster
 
+Deep generative model based forecasters
```

Comment:
should this be a new category with single inhabitant, or should we merge categories?

Reply:
We can merge it with the Transformer (deep learning) based forecasters. I would propose instead case to name it deep learning based forecasters. Would probably also fit better tk the LTFSDLinearForecaster, since I am unsure if they are transformers or not.

Reply:
Yes, makes sense. Perhaps separate PR to avoid clash with @yarnabrina PR?